### PR TITLE
chore(bundler): migrate from rollup to tsup

### LINKED ISSUE #66: [Feature]: make rc(release-candidate), beta release to publish experimental features
### **Package Scope**

- [ ] Add to an existing package
- [ ] New package

Package name:

### LINKED ISSUE #99: [Feature]: add tsup
### **Package Scope**

- [x] Add to an existing package
- [x] New package

Package name:

### PR DESCRIPTION
**Pull request by manudeli**
fix #99 
fix #66
related #101 

Co-authored-by: @okinawaa

* feat: bundler rollup to tsup

* feat(tsup): add @suspensive/tsup (#102)

* Create calm-deers-deliver.md

* fix(tsup): add multiple entry (#105)

---------

# migrate from rollup to tsup

I checked beta too after https://github.com/suspensive/react/pull/101 
We can check prerelease in [@suspensive/react@1.11.4-beta.0](https://github.com/suspensive/react/releases/tag/%40suspensive%2Freact%401.11.4-beta.0), [@suspensive/react-query@1.11.4-beta.0](https://github.com/suspensive/react/releases/tag/%40suspensive%2Freact-query%401.11.4-beta.0) before merging this

tsup migration is ready to be production release from prerelease(beta)

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

**Comment by okinawaa** [comment]
What should we verify in the prerelease phase !?

- Is the test code insufficient to verify stability?
- Or I think the preview feature of `vercel` is enough.

Can we check another thing with the [zip file](https://github.com/suspensive/react/releases/tag/%40suspensive%2Freact%401.11.4-beta.0)?

**Comment by manudeli** [comment]
> Can we check another thing with the [zip file](https://github.com/suspensive/react/releases/tag/%40suspensive%2Freact%401.11.4-beta.0)?

If we don't make this beta branch we also couldn't make this prerelease [zip file](https://github.com/suspensive/react/releases/tag/%40suspensive%2Freact%401.11.4-beta.0) because of missing `publishConfig.access: public`, so changeset publish will make error 403

@okinawaa 
So, first, as guided by the changeset prerelease, I created a branch with a separate `beta/tsup` branch from `main`. I could improve without missing 2 important things in this process.

1. I was able to add `publishConfig.access: public` to package.json to make it unblocked from npm when deploying from `main`. If there were other branches to merge into `main`, we can avoid a situation where these other branches can continue to receive this incorrect content due to `GitHub's update branch`. you can check this in [0138029](https://github.com/suspensive/react/pull/107/commits/0138029f238de3db87beb576c69ee68b9292b2f1)
2. We could do an `exit prerelease` of the changeset and merge. you can check this in [7fc7700](https://github.com/suspensive/react/pull/107/commits/7fc7700d5e5f04725872652d5c1a9c23e4f07540)

So we all did it to make safe new version of @suspensive/react, @suspensive/react-query with tsup, not rollup

Actually, this is my first beta release, thanks to your review, I was able to consider more things more carefully without any unnecessary BREAKING CHANGE. thank you so much.👍

File: .changeset/pre.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "mode": "pre",
+  "mode": "exit",
```
Comment (okinawaa) [inline .changeset/pre.json:2]: good!

**Comment by okinawaa** [comment]
>we can avoid a situation where these other branches can continue to receive this incorrect content 
👍🚀

**Review by okinawaa — APPROVED** [review]
Stability is so important as it's an open source, so thank you for your valuable work

**Pull request merged**